### ISSUE TITLE
Consider contributing to the Polymer CLI 🤗

### ISSUE THREAD

**Issue by justinfagnani** [aruntk/polymer-weight#2]
I've wanted this type of feature in the Polymer CLI for a while, but we haven't gotten around to it. Would you be willing to contribute to the CLI directly? If so, let's talk!

**Comment by aruntk** [comment]
@justinfagnani Yes. I'll be happy to contribute to the polymer-cli project. This is my handle on gitter https://gitter.im/aruntk
